src/lib/assets/yaml.ts

```diff
@@ -158,10 +175,24 @@ export async function overridesFile({ hash, name, image, config, apiToken }: Ass
         annotations: {},
       },
     },
+
+    // Custom RBAC section to add ClusterRoles and Roles
+    rbac: {
+      clusterRoles: customClusterRoleRules.map((rule, index) => ({
+        name: `custom-cluster-role-${index + 1}`,
```

Comment:
we need to use the rbac that we already create

Reply:
use singular role and clusterRole

Reply:
Changed to singular